<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1340</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Работники</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>800</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="deleteItemButton">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="addItemButton">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" rowspan="5" colspan="2">
        <widget class="QTableWidget" name="tableWidget">
         <column>
          <property name="text">
           <string>Ф.И.О.</string>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Возраст</string>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Высшее образов.</string>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Минимальная ЗП</string>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Желаемая должность</string>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
         </column>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Сортировать по:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Отчество</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Имя</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Высшее образование</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="confirmButton">
            <property name="text">
             <string>Подтвердить</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Дата рождения  </string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Фамилия</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QDateEdit" name="dateEdit"/>
          </item>
          <item row="6" column="1">
           <widget class="QDoubleSpinBox" name="salarySpinBox"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Желаемая должность</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEdit_3"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Минимальная ЗП</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lineEdit_4"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item row="5" column="1">
           <widget class="QRadioButton" name="radioButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Ранее занимаемые должности</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0" rowspan="5">
           <widget class="QListWidget" name="listWidget"/>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Добавить</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Должность</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit_5"/>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Стаж в месяцах</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="spinBox"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>Ф.И.О.</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Возраст</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Высшее образование</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Минимальная ЗП</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Желаемая должность</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="editItemButton">
         <property name="text">
          <string>Изменить</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="saveFileButton">
         <property name="text">
          <string>Сохранить файл</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="openFileButton">
         <property name="text">
          <string>Выбрать файл</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Вакансии</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="8" column="0">
        <widget class="QComboBox" name="comboBox_2">
         <item>
          <property name="text">
           <string>Название фирмы</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Должность</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Зароботная плата</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Длительность отпуска</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Опыт</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Необходимо высшего образования</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Возраст</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="editVocButton">
         <property name="text">
          <string>Изменить</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Сортировать по :                                   :</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2" rowspan="3">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="salaryVDoubleSpinBox"/>
          </item>
          <item row="7" column="1">
           <widget class="QSpinBox" name="minAgeSpinBox"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Зароботная плата</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="stageSpinBox"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Длительность отпуска</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Минимальный возраст</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QSpinBox" name="maxAgeSpinBox"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="positionVLineEdit"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="nameFirmLineEdit"/>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="vocationSpinBox"/>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Максимальный возраст</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Название фирмы</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Необходимо высшее образование</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Опыт (в годах)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Должность</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QRadioButton" name="radioButton_2">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="confirmVButton">
            <property name="text">
             <string>Подтвердить</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="openFile1Button">
         <property name="text">
          <string>Выбрать файл</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QPushButton" name="showCandidatesButton">
         <property name="text">
          <string>Вывести список возможных кандидатов</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="deleteVocButton">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>800</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="1" rowspan="2">
        <widget class="QListWidget" name="listWidget_2"/>
       </item>
       <item row="2" column="0" rowspan="5" colspan="2">
        <widget class="QTableWidget" name="tableWidget_2">
         <column>
          <property name="text">
           <string>Фирма</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Должность</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Зароботная плата</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Длительность отпуска</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Опыт (в годах)</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Нужно высшее обр.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Возраст</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="addVocButton">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="saveFile2Button">
         <property name="text">
          <string>Сохранить файл</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Страница</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>351</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Вывести дефицитные должности</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>351</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Вывести имена кандидатов, для которых не найдена вакансия</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget_3">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>20</y>
         <width>881</width>
         <height>611</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
